src/components/Textbook/WordDetail/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import Styles from './wordDetail.module.scss';
+import Button from '../../Shared/Buttons';
+import Play from '../../../../public/play.svg';
+
+interface wordDetailProps {
+  name: string;
+  synonyms: string;
+  phonetic: string;
+  definition: string;
+  example: string;
+  isLearnt: boolean;
+  inDictionary: boolean;
+  imgSrc: string;
+  encountered: number;
+  learned: number;
+  bestSeries: number;
+  handleLearnedToggle: (wordName: string) => void;
+  handleDictionaryToggle: (wordName: string) => void;
+}
+
+const WordDetail = (props: wordDetailProps) => {
+  return (
+    <div className={Styles.wordDetail}>
```

Comment:
You can destructure the incoming variable and use it directly
`const {name, synonyms, phonetic, definition, example, isLearnt, inDictionary, imgSrc, encountered, learned, bestSeries, handleLearnedToggle, handleDictionaryToggle} = props`
you don’t need to get the value through prop every time afterwards